docker/ProdDjangoDockerfile

```diff
@@ -4,39 +4,41 @@ FROM node:${NODE_VER} AS node-assets
 # Make npm output less verbose
 ENV NPM_CONFIG_LOGLEVEL warn
 
-# Workaround to avoid webpack hanging, see:
-# https://github.com/webpack/webpack-dev-server/issues/128
-ENV UV_THREADPOOL_SIZE 128
```

Comment:
If you've tested and the UV_THREADPOOL_SIZE override is not necessary for full builds, great! See comments in https://github.com/freedomofpress/securethenews/pull/198/commits/a0cddeb2a99e4dbc77d60919cb51f188def87cd5 ; I suspect it's worth preserving.

Reply:
I have added this back in my working branch. Removal was a regression.